### DISCUSSION TITLE
codebrowser书签插件

### DISCUSSION THREAD

**Discussion by giscus** [caibingcheng/comment#311] [Announcements]
# codebrowser书签插件

在使用codebrowser查阅一些源码的时候, 因为没有书签跳转功能, 有时候追踪调用栈就不是很方便. 因此, 开发了一款用于codebrowser书签功能的油猴插件.

https://imcbc.cn/202302/codebrowser-bookmark-script/

**Comment by penndu** [comment]
给大佬点个赞！
